python/nano/tutorial/training/pytorch-lightning/run_nano_pytorch_lightning_test.sh

```diff
@@ -0,0 +1,11 @@
+export ANALYTICS_ZOO_ROOT=${ANALYTICS_ZOO_ROOT}
+export NANO_HOME=${ANALYTICS_ZOO_ROOT}/python/nano/src
+export NANO_TUTORIAL_TEST_DIR=${ANALYTICS_ZOO_ROOT}python/nano/tutorial/training/pytorch-lightning/
+
+set -e
+
+sed -i s/Trainer\(max_epochs=5\)/Trainer\(max_epochs=5,\ fast_dev_run=True\)/ lightning_channel_last.py
```

Comment:
1. maybe you should use `$NANO_TUTORIAL_TEST_DIR/lightning_channel_last.py` to  replace `lightning_channel_last.py`
2. I noticed this file contains `Trainer(max_epochs=5, channels_last=True)`, so you should use sed to replace `Trainer(max_epochs=5, channels_last=True)`, not `Trainer(max_epochs=5)`